src/openloco/localisation/languages.cpp

```diff
@@ -0,0 +1,30 @@
+#include "languages.h"
+
+namespace openloco::localisation
+{
+    // clang-format off
+    const language_descriptor language_descriptors[languages::count] =
+    {
+        { "",       "",               "",                 loco_language_id::english_uk },
```

Comment:
I'm not a fan of having the language list included in the game itself. Would rather have some header in the Yaml files

Reply:
> Maybe hardcode language files inside the executable file? 

That's exactly what this code is doing. It'll be compiled into the executable. :wink:

Reply:
I doubt the performance of a ‘startup time’ check is so significant. It’s not like this will be done each frame.

Reply:
It depends mostly on how fast the yaml parser really is... If we're actually having it parse all complete language files, the overhead might add up to a significant amount. This warrants some experimenting, obviously.